src/content/tools/analysis.md

```diff
@@ -368,6 +373,74 @@ Due to YAML restrictions,
 You can use the other syntax for rules in an included file.
 :::
 
+## Including shared options
+
+An analysis options file can include other options which are specified another
```

Comment:
"specified another" --> "specified in another"

Reply:
Done